Zoo/src/ua/lviv/iot/algo/part1/lab1/Bank.java

```diff
@@ -0,0 +1,37 @@
+package ua.lviv.iot.algo.part1.lab1;
+
+import lombok.*;
```

Comment:
Never import * from side modules/libraries. This may cause a lot of name collision issues